package/MDAnalysis/core/topologyobjects.py

```diff
@@ -107,9 +107,14 @@ def type(self):
     def is_guessed(self):
         return bool(self._guessed)
 
+    # TODO: which one to use?
```

Comment:
@richardjgowers there are two hash functions in this clas. Which one is correct?